src/modules/BootStorage.rb

```diff
@@ -240,17 +249,24 @@ def boot_partitions
       stage1_partitions_for(boot_filesystem)
     end
 
+    # shortcut to get stage1 partitions for /
+    def root_partitions
```

Comment:
do you test it with some more fancy partitioning like having root on MD raid over LVM? As I am not sure if systemd boot wants RAID partition or physical one where it lives.

Reply:
Quite difficult to test all scenarios, isn't it ? Josef, how have you tested it ?

Reply:
well, usually it is tested by openQA and if I touch code that affects it then I give it a manual run with combination ( so usually LVM and software raid together ) and test if it works as expected.